modules/@apostrophecms/doc/index.js

```diff
@@ -987,9 +988,33 @@ module.exports = {
           'slug'
         ];
       },
+      // Add context menu operation to be used in AposDocContextMenu.
+      // Expected operation format is:
+      // {
+      //   context: 'update',
+      //   action: 'someAction',
+      //   modal: 'ModalComponent',
+      //   label: 'Context Menu Label'
+      // }
+      // All properties are required.
+      // The only supported `context` for now is `update`.
+      // `action` is the operation idefntifier and should be unique per a module -
```

Comment:
I see why you want to have an identifier. In that case please do not namespace it by module, and please allow over modules to override it by pushing one with the same action identifier (last module configured wins). This would be more typical of our patterns.